### DISCUSSION TITLE
copy settings to subdomains

### DISCUSSION THREAD

**Discussion by adrigon1503** [globaleaks/globaleaks-whistleblowing-software#3427] [Q&A]
Hi, I was wondering if there was a way for subdomains created from the web interface to apply the texts and settings without having to touch each subdomain separately?

**Comment by evilaliv3** [comment]
Thank you for your question @adrigon1503 

At the moment there is no possibility to do something similar to what you describe.

We are working on a concept of "platform profile" to make it possible to create costomizations profiles and instantiate them manually or automatically.

The main ideas are all annotated on this ticket: https://github.com/globaleaks/GlobaLeaks/issues/2039

 I invite you to read it and possibily add your suggestions on what you would like to be able to do.
